### ISSUE TITLE
Feature Request: Relative link paths on images

### ISSUE THREAD

**Issue by jmarkle825** [wvuweb/brand-patterns#53]
This is a feature request to change the absolute file paths for images in the out of the box css to be relative file paths to alleviate additional DOM level requests. Currently, everything is linked directly to the files in the patterns site, "https://patterns.wvu.edu/images/flying-wv.png" rather than "../images/flying-wv.png"

This would just help with optimization. Not a ton since only a few things really get called in to any given site, so this isn't high level at all. I feel it could be a smart move in the long run though because it would also prevent a bunch of stuff breaking if you're working on patterns and it crashes, or something terrible happens and someone commits a bunch of code that breaks a bunch of other random stuff.

**Comment by adamjohnson** [comment]
This is related to #38.

Right now we're doing this so that we don't have 4 billion `flying-wv.png` files in every single repo in Stash. It also helps us accommodate any changes to any of those files that may be needed.

At the moment, the `images/ie8` folder houses a duplicate of all those files and they are strewn everywhere across our Stash CleanSlate theme repos. Since we're no longer supporting IE8 I could see us removing that folder altogether. 

Basically, it comes down to a management issue. We prefer to have those assets centrally located. And, once #38 is closed, those images will likely load fairly quickly. Not to mention static.wvu.edu is a likely candidate to move to Amazon S3 and be CDN-ified sooner rather than later.
